test/Microsoft.AspNetCore.Authentication.Test/JwtBearer/JwtBearerMiddlewareTests.cs

```diff
@@ -215,67 +183,37 @@ public class JwtBearerMiddlewareTests
         [Fact]
         public async Task RetrievingTokenFromAlternateLocation()
         {
-            var server = CreateServer(new JwtBearerOptions
+            var options = new JwtBearerOptions()
             {
                 AutomaticAuthenticate = true,
                 Events = new JwtBearerEvents()
                 {
-                    OnReceivingToken = context =>
+                    OnMessageReceived = context =>
                     {
                         context.Token = "CustomToken";
                         return Task.FromResult<object>(null);
-                    },
-                    OnReceivedToken = context =>
```

Comment:
I'm fine. This middleware was totally inappropriate for simple token validation since @brentschmaltz's last year changes anyway.